### PR TITLE
Move chef-core to the foundation project

### PR DESCRIPTION
**Pull request by tas50**
This is a foundation project.

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

**Comment by tas50** [comment]
ping @chef/chef-foundation-reviewers

File: projects/chef-foundation.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ Tyler Ball
 
 - [appbundle-update](https://github.com/chef/appbundle-update)
 - [appbundler](https://github.com/chef/appbundler)
+- [chef-core](https://github.com/chef/chef-core)5
```
Comment (scotthain) [inline projects/chef-foundation.md]: Extra `5` here?

Comment (tas50) [reply projects/chef-foundation.md]: :(

Comment (tas50) [reply projects/chef-foundation.md]: Fixed

**Pull request merged**